website/content/en/docs/contribution-guidelines/plugins.md

```diff
@@ -55,7 +55,7 @@ Note that custom plugins are called via the init sub-command to work as global p
 
 ## Plugins Vision
 
-Maintainers and contributors are able to create their own plugin(s) using the same standards and approaches. These contributions in the future could be added as solutions to be compatible and used with the Operator SDK CLI. Currently, the SDK cannot discover and use plugins that are not shipped or built with its binary. However, it has been discussed in the issue [kubebuilder/issues/1378][kb-issue].
+Maintainers and contributors are able to create their own plugin(s) using the same standards and approaches. By following it, that seems that could facilitate any future option for these contributions to be compatible and used with the Operator SDK CLI if possible and might also reduce the effort required to include them in this project via a donation process if acceptable. Currently, the SDK cannot discover and use plugins that are not shipped or built with its binary. However, it has been discussed in the issue [kubebuilder/issues/1378][kb-issue].
```

Comment:
You mean, it would be easier for operator-sdk to accept a plugin if the devs follow this guide and adopt the standards/approaches outlined here?

Reply:
Not easy to accept but easy to actually add the solution. 

Example:

I wrote a solution to scaffold python operators then, I want to donate this solution to SDK (assuming that is desirable by SDK). If that not follow up its standards that means that we will need first to find a way to align the solution with the SDK layout standards to see if that is possible and might the effort not justify that anymore. 

However, if I develop my python solution as SDK does ansible/helm then that would be easier to actually be added to the project.